using-mobile-extensions/adobe-target/README.md

```diff
@@ -506,9 +382,14 @@ If you need to update SDK configuration, programmatically, use the following inf
 | target.environmentId | Environment ID you want to use, if this is left blank, the default production environment will be used. |
 | target.propertyToken | `at_property token` value, which is generated from the Target UI. If this value is left blank, no token is sent in the Target network calls. |
 | target.previewEnabled | Boolean parameter, which can be used to enable/disable Target Preview. If not specified, then Preview will be enabled by default. |
+| target.sessionTimeout | Time, in seconds, for which the Target session ID and Egde Host are persisted. If not specified, then default timeout value is 30 minutes. |
```

Comment:
Please reword the description as follows:
"The duration, in seconds, during which the Target session ID and Egde Host are persisted. If this value is not specified, the default timeout value is 30 minutes."